### DISCUSSION TITLE
[Improve] Make an option to determine particular recipients of a stock level email notifications

### DISCUSSION THREAD

**Discussion by Caleydon** [PrestaShop/PrestaShop#34215] [Ideas]
### The problem

Hi. This is a request for improving existing feature.

**PrestaShop allow sending email notification for all employee when stock of the product is lower than set value.**

Issue

As PrestaShop developer I find out that I receive this type of stock level email notifications from client's eshops, because I've got full admin BO account in many eshops that I maintain. 

As maintainer/developer I need to have access in all areas of PrestaShop including stock. It's a pretty annoying to receive those type of emails that belongs merchants only. ALso eshop could have many employee accounts but there is maybe only one that is responsible for eshop's stock level. But notifications are receiveding all that have access into the stock.

A few eshop owners ask me to disable this feature because they have employees that maintain stock but as eshop owner need to have full access account.

![stock-level-notifications](https://github.com/PrestaShop/PrestaShop/assets/2806752/d226b9be-e8fa-46d4-94e1-1725a42b387e)

### The solution

Could you please add an option in the employee account that will enable/disable to receive those notifications? 

![add-option-receive-notifications](https://github.com/PrestaShop/PrestaShop/assets/2806752/63cea075-9acb-4bca-833f-72397c844d18)

### Alternatives

Maybe you will find better solution.

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
That would be great. The current behavior is quite annoying.

**Comment by kpodemski** [comment]
WDYT @MatShir ?

**Comment by MatShir** [comment]
Very good idea ! @Caleydon, if the user remove its, we should remove them from the config of ps_emailalert.

@kpodemski If a remember well, the other issue for merchant notification system are composed of ps_emailalert module and the core software. We have recurrent report of users still receiving info without really understanding where it comes from.
IMO we should deport all the behavior in the module. So we can easily improve and one place to configure.
